### ISSUE TITLE
Staging Review finding: Focus lost when landing on the "Your benefits and resources" results page

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#93696]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Transition Experience
**Product:** Transition Experience MVP v1
**Feature:** V1 Transition Experience

## Findings details
**VA.gov Experience Standard - issue:** [User experiences an unexpected interaction with a component or pattern.](https://design.va.gov/about/experience-standards/#user-experiences-predictable-interactions-with-components)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

When submitting your entries and landing on the "Your benefits and resources" page to review your results, keyboard focus is lost. For keyboard-only users, it's not too difficult to recover from this. For screen reader users, nothing is announced. Screen reader users are left to guess at what happened --- Did it work? Is it broken? --- and have to explore the page to determine where they are.

<details><summary>Link, screenshot or steps to recreate</summary>

Steps to reproduce:
1. Navigate to the review page.
2. Activate a screen reader. (Easiest options for testing: VoiceOver on Mac or NVDA on Windows.)
3. Press [tab] until focus moves to the "Submit" button.
4. Press [enter] to activate the button..
5. Note that your screen reader does not announce anything after you press "Submit."

</details>

## Recommended action

This is one of those things you get for free with plain vanilla HTML, but in a single-page application you have to think about how you'll manage focus in these transitions. The forms library does a lot of the work, but since you're doing something a little different after "Submit" you'll need to adjust the focus management here.

What we would typically expect on VA.gov is that when landing on the "Your benefits and resources" results page focus goes to the H1 using `tabindex="-1"` to assign focus.

## References

- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by nkontrabecki** [comment]
Confirmed fixed and tested in Sprint 14.
